tests/test_samples.py

```diff
@@ -407,7 +402,7 @@ def test_plot_1d_colours():
 def test_astropyhist():
     np.random.seed(3)
     ns = NestedSamples(root='./tests/example_data/pc')
-    ns.plot_1d(['x0', 'x1', 'x2', 'x3'], kind='hist_1d', plotter='astropyhist')
```

Comment:
Might some people want to use the astropyhist plotter for non-knuth based reasons? Should we test both?

Reply:
I think the only added functionality of astropyhist is in `'knuth'`, `'freedman'` and `'blocks'`, which are all tested here. Note that it is the `bins` arg that indicates the use of astropyhist.